JSQMessagesViewController/Layout/JSQMessagesBubblesSizeCalculator.h

```diff
@@ -29,12 +29,14 @@
 /**
  *  Initializes and returns a bubble size calculator with the given cache and minimumBubbleWidth.
  *
- *  @param cache              A cache object used to store layout information.
- *  @param minimumBubbleWidth The minimum width for any given message bubble.
+ *  @param cache                        A cache object used to store layout information.
+ *  @param minimumBubbleWidth           The minimum width for any given message bubble.
+ *  @param usesFixedWidthMessageBubbles Boolean indicating if you want same width bubbles in landscape & portrait
  *
  *  @return An initialized `JSQMessagesBubblesSizeCalculator` object if successful, `nil` otherwise.
  */
 - (instancetype)initWithCache:(NSCache *)cache
-           minimumBubbleWidth:(NSUInteger)minimumBubbleWidth NS_DESIGNATED_INITIALIZER;
+		   minimumBubbleWidth:(NSUInteger)minimumBubbleWidth
+		usesFixedWidthBubbles:(BOOL)fixedWidthMessageBubbles NS_DESIGNATED_INITIALIZER;
```

Comment:
1. spacing/tabs
2. parameter name and variable name should be the same
~~`(BOOL)fixedWidthMessageBubbles`~~ `(BOOL)usesFixedWidthBubbles `